### ISSUE TITLE
why some body said there should not be any batchnorm in Unet? 

### ISSUE THREAD

**Issue by chenxingshenSecond** [milesial/Pytorch-UNet#216]
But this implementation had batchnorm after most conv?

**Comment by milesial** [comment]
This is a customized implementation for the Carvana challenge. Part of the customization includes adding batchnorm layers.
